### PR TITLE
pk-task: Modernize the codebase

### PR DESCRIPTION
**Pull request by tintou**
No functional change but use the latest GObject coding standards.

### CODE REVIEW

**Comment by sidt4** [comment]
Thanks for this PR!

I'm not sure, but I guess we should resolve https://github.com/PackageKit/PackageKit/issues/709 before making new code changes in that area.

**Comment by tintou** [comment]
> Does this raise the GLib minimum version?

It doesn't

**Comment by tintou** [comment]
> Thanks for this PR!
> 
> I'm not sure, but I guess we should resolve #709 before making new code changes in that area.

I'll try to look into this but this shouldn't block any change

File: lib/packagekit-glib2/pk-task.c [hunk 1]
```diff
@@ -58,13 +56,15 @@ struct _PkTaskPrivate
 enum {
 	PROP_0,
 	PROP_SIMULATE,
-	PROP_ONLY_PREPARE,
+	PROP_ONLY_DOWNLOAD,
```
Comment (Conan-Kudo) [inline lib/packagekit-glib2/pk-task.c:59]: Why was this renamed?

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

File: lib/packagekit-glib2/pk-task.c (hunk 1 above)
Comment (tintou) [reply lib/packagekit-glib2/pk-task.c:59]: The property is actually named "only-download" and the setter/getter are named set/get_only_download so It makes sense to have it consistent here too

Comment (tintou) [reply lib/packagekit-glib2/pk-task.c:59]: This has no implication on API/ABI

**Review by Conan-Kudo — APPROVED** [review]

File: lib/packagekit-glib2/pk-task.c (hunk 1 above)
Comment (Conan-Kudo) [reply lib/packagekit-glib2/pk-task.c:59]: I didn't think it did, since you didn't change the order in the enumeration.
